cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -431,6 +463,21 @@ void CobaltContentBrowserClient::CreateFeatureListAndFieldTrials() {
   // instance is set.
   feature_list->RegisterExtraFeatureOverrides(feature_overrides);
 
+  std::unique_ptr<variations::SeedResponse> initial_seed;
+
+  CobaltVariationsServiceClient variations_service_client;
+  variations::VariationsFieldTrialCreator field_trial_creator(
+      &variations_service_client,
+      std::make_unique<variations::VariationsSeedStore>(
+          GlobalFeatures::GetInstance()->experiment_config(),
+          std::move(initial_seed),
```

Comment:
nit: Or, just do here `std::make_unique<variations::SeedResponse>()` ?
I see the benefit in having a name to this (`initial_seed`) but since
it's just an empty object, I'd lean towards conciseness.